### ISSUE TITLE
Resetting Flight will not allow a Play Track to accompany it.

### ISSUE THREAD

**Issue by icedomin8r** [opentx/opentx#1546]
Didn't see this on the list of issues, apologies if this is a repeat.

Running Companion 2.0.7 and Taranis FW opentx-taranis-haptic-en-2.0.7

I have all my models set up to reset various parameters depending on the aircraft. I do this by setting up a logical switch to Edge, assign V1 to SH↓ and V2 to 0.5. I can then use this to reset whatever.

Right under the special function I set up for the reset I set up another function to Play Track "reset" as well.

It will speak "reset" as long as the reset special function is not set to Flight. 

It shows under the special function list that the logical switch is being activated, but no sound is played. Having a Play Sound, or a Play Haptic response works, just not a Play Track.

Basically this means you need to set up more special functions to reset more than one parameter if you want a Play Track to fire.

Not a big deal, just a heads up.
